torchgen/model.py

```diff
@@ -1148,7 +1148,11 @@ def kind(self) -> SchemaKind:
             return SchemaKind.functional
 
     def signature(
-        self, *, strip_default: bool = False, strip_view_copy_name: bool = False
+        self,
+        *,
+        strip_default: bool = False,
+        convert_mutable_inputs_to_returns: bool = False,
+        strip_view_copy_name: bool = False,
```

Comment:
this is my "too many kwargs" grumpy face

Reply:
:'(